src/modules/flow/iio/nodes.c

```diff
@@ -776,12 +776,16 @@ humidity_open(struct sol_flow_node *node, void *data, const struct sol_flow_node
 }
 
 static bool
-adc_create_channels(struct iio_double_data *mdata, int device_id)
+adc_create_channels(struct iio_double_data *mdata, int device_id, int channel_id)
 {
+    char channel_name[NAME_MAX];
+
+    snprintf(channel_name, sizeof(channel_name), "in_voltage%d", channel_id);
```

Comment:
Thanks @ceolin for the snprintf, I found there are several different places in this nodes.c files, let me add the error checking in another PR